buildspecs/release-javadoc.yml

```diff
@@ -22,3 +22,5 @@ phases:
     - aws s3 sync $DOC_PATH/$RELEASE_VERSION/ $DOC_PATH/latest/ --acl=public-read --delete
     - jar cf aws-java-sdk-v2-docs.jar -C target/site/apidocs .
     - aws s3 cp aws-java-sdk-v2-docs.jar $DOC_PATH/ --acl="public-read"
+    - python ./scripts/doc_crosslinks/generate_cross_link_data.py --apiDefinitionsBasePath ./services/ --apiDefinitionsRelativePath src/main/resources/codegen-resources/service-2.json  --templatePath ./scripts/doc_crosslinks/crosslink_redirect.html --outputPath ./scripts/crosslink_redirect.html
```

Comment:
Are there any alternative folders for the source of the template html file and the output? Especially the output doesn't seem like it belongs in the scripts folder, but if there is no other logical place to put them, it may be unnecessary to create a new folder.

Reply:
There are no alternatives fir template html.
The "doc_crosslinks" is created to make sure all the required scripts/resource can be found in a single place.
The processed html is copied to parent ./script folder when the script is executed as a part of build process this output file is a temp file which is immediately copied to S3 .